components/Buttons/src/MDCButton.m

```diff
@@ -299,6 +299,7 @@ - (void)layoutSubviews {
   } else {
     _inkView.frame = self.bounds;
   }
+  self.titleLabel.frame = MDCRectAlignToScale(self.titleLabel.frame, [[UIScreen mainScreen] scale]);
```

Comment:
Same about scale, tho I know it's inconsistent in the codebase. We can correct them as we go along or feel free to do a PR with a batch change. Same for bounds.